packit_service/worker/handlers/distgit.py

```diff
@@ -488,6 +488,20 @@ def _report_errors_for_each_branch(self, message: str):
     def get_resolved_bugs(self):
         raise NotImplementedError("Use subclass.")
 
+    @staticmethod
+    def report_dashboard_url(
+        sync_release_pull_request: SyncReleasePullRequestModel,
+        downstream_pr: PullRequest,
+        dashboard_url: str,
+    ):
+        msg = f"Logs and details of the syncing: [Packit dashboard]({dashboard_url})"
+        # this is a retrigger
+        if len(sync_release_pull_request.sync_release_targets) > 1:
+            downstream_pr.comment(msg)
+        else:
+            original_title = downstream_pr.title
+            downstream_pr.title = original_title + "\n---\n" + msg
```

Comment:
```suggestion
            downstream_pr.title = original_title + "\n---\n" + msg
```

I will be surprised if we can do this operation.
I don't think we have the permissions to update our own PR, at least not until I finish my work on the Pagure side and I create a new token.

And I am also wondering if this is not too long as a title? 🤔

Reply:
But we can try it on stg, to be 100% sure.

Reply:
I am dumb :D it was meant to be `description`. But yet, you are right, I am getting exc because of insufficient ACLs. I will have a look into it more, but I wanted to avoid having this logic directly in Packit as it is service-related. In case I won't find a way to do this nicely, I would wait for your Pagure changes to be deployed for this as well. Thanks for pointing this out!